src/server/qgscapabilitiescache.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include <QCoreApplication>
 #include <QFileInfo>
+#include <QDir>
```

Comment:
gotta make the new include sort script happy :)
```suggestion
#include <QDir>
#include <QFileInfo>
```